src/view/logging_mode/dmx_data_log.py

```diff
@@ -0,0 +1,107 @@
+# coding=utf-8
+"""Widget for displaying DMX current data"""
+
+from PySide6 import QtCore, QtWidgets
+from PySide6.QtCore import QTimer
+
+from model import Broadcaster, PatchingUniverse
+from model.final_globals import FinalGlobals
+from model.patching_channel import PatchingChannel
+from proto.DirectMode_pb2 import dmx_output
+
+
+class DmxDataLogWidget(QtWidgets.QWidget):
+    """Widget to Log DMX Data"""
+
+    def __init__(self, broadcaster: Broadcaster):
+        super().__init__()
+        self.setGeometry(self.geometry().x(), self.geometry().y(), 150, FinalGlobals.get_screen_height())
+        self._broadcaster = broadcaster
+        self._timer = QTimer()
+        self._timer.setInterval(1000)
+        self._timer.timeout.connect(self._request_dmx_data)
+
+        self._widgets = QtWidgets.QTabWidget(self)
+        self._universes: list[tuple[PatchingUniverse, list[DmxLogItem]]] = []
+
+        layout = QtWidgets.QVBoxLayout()
+        layout.addWidget(self._widgets)
+        self.setLayout(layout)
+
+        self._broadcaster.add_universe.connect(self.react_add_universe)
```

Comment:
We should make sure that this widget also behaves the correct way if the constructor was called after universes were already added.

Reply:
i see it differently all elements that listen to the broadcaster should be created before any broadcast comes otherwise there are huge amounts of code duplicates or very unnecessary code fragments

Reply:
One could just put a loop in the constructor, in addition to the signal linking: for every already existing universe call _add_universe as well. But I'll merge this as-is now.